lib/ansible/modules/network/fortios/fortios_voip_profile.py

```diff
@@ -0,0 +1,1152 @@
+            notify-rate: "78"
+            open-contact-pinhole: "disable"
+            open-record-route-pinhole: "disable"
+            open-register-pinhole: "disable"
+            open-via-pinhole: "disable"
+            options-rate: "83"
+            prack-rate: "84"
+            preserve-override: "disable"
+            provisional-invite-expiry-time: "86"
+            publish-rate: "87"
+            refer-rate: "88"
+            register-contact-trace: "disable"
+            register-rate: "90"
+            rfc2543-branch: "disable"
+            rtp: "disable"
+            ssl-algorithm: "high"
+            ssl-auth-client: "<your_own_value> (source user.peer.name user.peergrp.name)"
+            ssl-auth-server: "<your_own_value> (source user.peer.name user.peergrp.name)"
+            ssl-client-certificate: "<your_own_value> (source vpn.certificate.local.name)"
+            ssl-client-renegotiation: "allow"
+            ssl-max-version: "ssl-3.0"
+            ssl-min-version: "ssl-3.0"
+            ssl-mode: "off"
+            ssl-pfs: "require"
+            ssl-send-empty-frags: "enable"
+            ssl-server-certificate: "<your_own_value> (source vpn.certificate.local.name)"
+            status: "disable"
+            strict-register: "disable"
+            subscribe-rate: "106"
+            unknown-header: "discard"
+            update-rate: "108"
+'''
+
+RETURN = '''
+build:
+  description: Build number of the fortigate image
+  returned: always
+  type: str
+  sample: '1547'
+http_method:
+  description: Last method used to provision the content into FortiGate
+  returned: always
+  type: str
+  sample: 'PUT'
+http_status:
+  description: Last result given by FortiGate on last operation applied
+  returned: always
+  type: str
+  sample: "200"
+mkey:
+  description: Master key (id) used in the last call to FortiGate
+  returned: success
+  type: str
+  sample: "id"
+name:
+  description: Name of the table used to fulfill the request
+  returned: always
+  type: str
+  sample: "urlfilter"
+path:
+  description: Path of the table used to fulfill the request
+  returned: always
+  type: str
+  sample: "webfilter"
+revision:
+  description: Internal revision number
+  returned: always
+  type: str
+  sample: "17.0.2.10658"
+serial:
+  description: Serial number of the unit
+  returned: always
+  type: str
+  sample: "FGVMEVYYQT3AB5352"
+status:
+  description: Indication of the operation's result
+  returned: always
+  type: str
+  sample: "success"
+vdom:
+  description: Virtual domain used
+  returned: always
+  type: str
+  sample: "root"
+version:
+  description: Version of the FortiGate
+  returned: always
+  type: str
+  sample: "v5.6.3"
+
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+fos = None
+
+
+def login(data):
+    host = data['host']
+    username = data['username']
+    password = data['password']
+
+    fos.debug('on')
+    if 'https' in data and not data['https']:
```

Comment:
can be modified as:
```suggestion
    fos.https('off') if 'https' in data and not data['https'] else fos.https('on')
```

Reply:
I am a big fan of oneliners so I really like this suggestion. However we decided to do it the other way because we are always receiving comments about readability and avoid oneliners when possible. If it is not a showstopper could we leave it as it is?